content/applications/hr/recruitment/add-new-applicants.rst

```diff
@@ -0,0 +1,169 @@
+  :icon:`fa-angle-right` :guilabel:`(right)` arrows on either side of the month to navigate to the
+  desired month, then click the date. Leaving this field blank indicates the applicant can start
+  immediately.
+- :guilabel:`Tags`: select as many tags as desired from the drop-down menu. To add a tag that does
+  not exist, type in the tag name, then click :guilabel:`Create "new tag"`.
+
+Job section
+-----------
+
+The following fields are pre-populated when creating a new applicant, as long as these field are
+specified on the Job Position. Editing the fields is possible, if desired.
+
+- :guilabel:`Applied Job`: select the job position the applicant is applying to from the drop-down
+  menu.
+- :guilabel:`Department`: select the department the job position falls under from the drop-down
+  menu.
+- :guilabel:`Company`: select the company the job position is for using the drop-down menu. This
+  field only appears when in a multi-company database.
+
+Contract section
+----------------
+
+- :guilabel:`Expected Salary`: enter the amount the applicant is requesting for the role in this
+  field. The number should be in a `XX,XXX.XX` format. The currency is determined by the
+  localization setting for the company.
+- :guilabel:`Extra advantages...`: if any extra advantages are requested by the applicant, enter it
+  in the :guilabel:`Extra advantages...` field to the right of the :guilabel:`Expected Salary`
+  field. This should be short and descriptive, such as `1 week extra vacation` or `dental plan`.
+- :guilabel:`Proposed Salary`: enter the amount to be offered to the applicant for the role in this
+  field. The number should be in a `XX,XXX.XX` format.
+- :guilabel:`Extra advantages...`: if any extra advantages are offered to the applicant, enter it in
+  the :guilabel:`Extra advantages...` field to the right of the :guilabel:`Proposed Salary` field.
+  This should be short and descriptive, such as `unlimited sick time` or `retirement plan`.
+
+Application summary tab
+-----------------------
+
+Any additional details or notes that should be added to the applicant's card can be typed into this
+field.
+
+Skills tab
+----------
+
+Skills can be added to the applicant's card. For details on adding skills, refer to the
+:ref:`Create new employees <employees/skills>` document.
+
+.. image:: add-new-applicants/new-applicant.png
+   :align: center
+   :alt: All the fields for a new applicant form entered.
+
+.. _recruitment/quick-add-applicant:
+
+Quick add
+=========
+
+Navigate to the main recruitment dashboard by going to :menuselection:`Recruitment app -->
+Applications --> By Job Positions`. Click the :guilabel:`(#) New Applications` button on the job
+position card that the applicant should be added to. Then, click the on the :guilabel:`Quick Add`
+button, represented by a small :icon:`fa-plus` :guilabel:`(plus)` icon in the top-right corner of
+the :guilabel:`New` stage to quickly add a new applicant.
```

Comment:
```suggestion
Navigate to the main recruitment dashboard by going to :menuselection:`Recruitment app -->
Applications --> By Job Positions`. 

From the :guilabel:`Job Positions` dashboard, click the :guilabel:`(#) New Applications` button on the job
position card that the applicant should be added to.

Then, from the :guilabel:`Applications` page that appears, click the on the :guilabel:`Quick Add` button, represented by a small :icon:`fa-plus` :guilabel:`(plus)` icon in the top-right corner of the :guilabel:`New` stage to quickly add a new applicant.

Doing so reveals a blank Kanban card with various fields to fill out.
```
I'm a bit conflicted as to whether or not the 'Quick Add' button should be used with a GUI here... 🤔 It's not shown on the UI, so it might be better to rephrase this in a way, like, "Then, from the 'Applications' page, quickly add a new applicant by clicking the 'fa-plus' (plus) icon in the top-right..."

Whatever you think is best...

Reply:
I had a hard time with this, because I wanted the word 'Quick Add' to be called out since that's the name of the seciton, AND that's what you see when you rollover on the image. Because the thing you click (the plus sign) IS on the screen- but it's actual name is the 'Quick Add' button. I am also torn- I think we need @samueljlieber to be the tie breaker and make a decision for us =D